.circleci/config.yml

```diff
@@ -28,9 +28,9 @@ jobs:
     docker:
       - image: cimg/node:16.11
     steps:
+      - checkout
       - vault/get-secrets: # Loads vault secrets
           template-preset: 'semantic-release'
```

Comment:
after this is merged, we should make a new PR that changes this to `semantic-release-ecosystem`